.gitattributes

```diff
@@ -1,5 +1,5 @@
 # Define the line ending behavior of the different file extensions
-# Set default behavior, in case users don't have core.autocrlf set.
+# Set default behaviour, in case users don't have core.autocrlf set.
```

Comment:
Look one line above too

Reply:
American English always for documentation. So behavior and color etc is correct.

Reply:
I flipped it back after merge.